ProjFS.Mac/PrjFSKextLogDaemon/PrjFSKextLogDaemon.cpp

```diff
@@ -196,11 +203,7 @@ static void StartLoggingKextMessages(io_connect_t connection, io_service_t prjfs
     });
     dispatch_resume(logDataQueue->dispatchSource);
 
-    dispatch_source_t timer = nullptr;
-    if (TryFetchAndLogKextHealthData(connection))
```

Comment:
Is it worth saving the result of `TryFetchAndLogKextHealthData ` the first time it's called?  If so maybe we don't call it in the next interval to match the previous functionality?  

It looks like we no longer use the bool result of `TryFetchAndLogKextHealthData`

Reply:
> If so maybe we don't call it in the next interval to match the previous functionality?

@jeschu1 I think the previous behavior was actually less than ideal, because we'd call `TryFetchAndLogKextHealthData` here and then immediately again when the timer in `StartKextHealthDataPolling` started.

The old idea had been that if we're unable to fetch the health data there is no need to start the timer, but I realized while working on this we also use the timer to re-connect to the kext if there are connection issues, and so I *think* it's OK to keep trying every 30 minutes if the first attempt fails.  Let me know if you can think of any problems with this new approach.

> It looks like we no longer use the bool result of TryFetchAndLogKextHealthData

Nice catch, I will clean this up and see what should be renamed.